cookbooks/continental_compression_with_imposed_faults/doc/continental_compression_with_imposed_faults.md

```diff
@@ -0,0 +1,56 @@
+
+The cookbook builds directly on components of the {ref}`sec:cookbooks:crustal-deformation` cookbook, and the primary goal here is to:
+1. Highlight techniques for imposing fault zones in the initial conditions of a lithospheric deformation simulation and
+2. Demonstrate their effect on the evolution of deformation patterns.
+
+# Model Design
+The model domain spans 400 km x 400 km and uses adaptive refinement to resolve deformation patterns in the regions where faults are imposed at the onset of deformation. The initial thermal structure follows a conductive, continental-style geothermal through the lithosphere and an initial adiabatic profile in the asthenosphere. The governing equations follow the extended Boussinesq approximation, which includes both adiabatic and shear heating.
+
+Deformation is driven by horizontal velocity applied at the model sides (2 mm/yr), which are balanced by outflow at the model base. A free surface allows topography to develop through time, which is diffused at each time step to approximate landscape evolution processes and stabilize both linear and nonlinear solver behavior.
+
+The initial lithological structure includes distinct layers representing the upper crust (20 km), lower crust (20 km), mantle lithosphere (40 km), and asthenosphere (320 km). Each layer is represented by a nonlinear viscoplastic rheology combining dislocation creep and pressure-dependent plasticity. Respectively, the pre-yield viscosity and brittle material parameters (cohesion, friction) weaken by a factor of 10 and 4 as a function of accumulated viscous and brittle strain over strain intervals of 0.5-1.5.
+
+```{literalinclude} strain_weakening_mechanism.part.prm
+```
+
+Faults are correspondingly integrated into the model initial conditions as zones of initial strain defined in the Geodynamic World Builder. Four faults extending to the base of the lithosphere with constant dip angles are included, two of which dip at 60 degrees toward each other in a style representing pre-existing normal faults. A third vertical fault (i.e., approximating a strike-slip fault) is centrally located between the two dipping faults, while the fourth fault is located to the right of two normal faults and dips at a higher angle. The faults maintain a constant width of 5 km and extend to a depth of 100 km. In addition to defined fault locations with constant strain values, randomized zones of plastic (0-40 km depth) and viscous strain (0-100 km depth) are imposed across a 300 km wide zone in the upper to approximate pervasive off-fault damage observed in many regions that have undergone significant tectonic deformation. The plastic and viscous strain compositions value within the faults is the sum of the value defined in the world builder and the randomized values defined in these zones.
```

Comment:
I would also just add the x location of the faults (at the surface), and how deep the faults extend.